src/test/regress/expected/shard_rebalancer.out

```diff
@@ -181,6 +181,8 @@ RESET citus.shard_count;
 RESET citus.shard_replication_factor;
 RESET citus.replication_model;
 -- Create a user to test multiuser usage of rebalancer functions
+-- We explicitely don't create this user on worker nodes yet, so we can
+-- test some more error handling. We create them later there.
```

Comment:
this might require some extra care when merging to enterprise as the user will be created there via the propagation of objects when you use them.

Given that you only added the comment it might be the case that this has already been taken care of.